library_generation/test/model/config_change_unit_tests.py

```diff
@@ -0,0 +1,224 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+
+from library_generation.model.config_change import ChangeType
+from library_generation.model.config_change import ConfigChange
+from library_generation.model.config_change import LibraryChange
+from library_generation.model.gapic_config import GapicConfig
+from library_generation.model.generation_config import GenerationConfig
+from library_generation.model.library_config import LibraryConfig
+
+
+class ConfigChangeTest(unittest.TestCase):
+    def test_get_changed_libraries_with_repo_level_change_returns_none(self):
+        config_change = ConfigChange(
+            change_to_libraries={
+                ChangeType.REPO_LEVEL_CHANGE: [],
+                # add a library level change to verify this type of change has
+                # no impact on the result.
+                ChangeType.LIBRARY_LEVEL_CHANGE: [
+                    LibraryChange(
+                        changed_param="test",
+                        latest_value="test",
+                        library_name="test-library",
+                    )
+                ],
+            },
+            baseline_config=ConfigChangeTest.__get_a_gen_config(),
+            latest_config=ConfigChangeTest.__get_a_gen_config(),
+        )
+        self.assertIsNone(config_change.get_changed_libraries())
```

Comment:
Similar to one of the comments above, the test would be a bit more readable if we added something like `self.assertEquals(ALL_LIBRARIES_AFFECTED, config_change.get_changed_libraries())`

Reply:
Done.